build-native.sh

```diff
@@ -31,6 +31,15 @@ while :; do
         linux-x64)
             _OSDir=linux-x64
             ;;
+        linux-x86)
+            _OSDir=linux-x86
+            ;;
+        linux-arm64)
+            _OSDir=linux-arm64
+            ;;
+        linux-arm)
+            _OSDir=linux-arm
+            ;;
```

Comment:
Is that it? Just set `_OSDir` and it will build with the correct architecture? I'm not sure that's the case?

Reply:
Oh, I forgot to add the build steps for cross compiling with Ubuntu Arm images on Docker. Thanks for the heads up.